ui/app/AppLayouts/WalletV2/components/CollectibleCollection.qml

```diff
@@ -41,157 +25,75 @@ Item {
-                anchors.verticalCenter: parent.verticalCenter
-                anchors.horizontalCenter: parent.horizontalCenter
                 width: 20
                 height: 20
+
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.horizontalCenter: parent.horizontalCenter
             }
         }
     }
 
     Component {
         id: loaded
-        ScrollView {
-            height: contentRow.height
+
+        Flow {
             width: parent.width
-            ScrollBar.vertical.policy: ScrollBar.AlwaysOff
-            ScrollBar.horizontal.policy: ScrollBar.AsNeeded
-            clip: true
-
-            Row {
-                id: contentRow
-                bottomPadding: Style.current.padding
-                spacing: Style.current.padding
-
-                Repeater {
-                    model: walletV2Model.collectiblesView.getAssetsList(root.slug)
-
-                    Item {
-                        width: image.width
-                        height: image.height
-                        clip: true
-
-                        RoundedImage {
-                            id: image
-                            width: 164
-                            height: 164
-                            border.width: 1
-                            border.color: Style.current.border
-                            radius: 16
-                            source: model.imageUrl
-                            fillMode: Image.PreserveAspectCrop
-                            anchors.verticalCenter: parent.verticalCenter
-                            anchors.horizontalCenter: parent.horizontalCenter
-                            onClicked: {
-                                collectibleModal.openModal({
-                                    name: model.name,
-                                    collectibleId: model.id,
-                                    description: model.description,
-                                    permalink: model.permalink,
-                                    imageUrl: model.imageUrl
-                                })
-                            }
+
+            bottomPadding: 16
+            spacing: 24
+
+            Repeater {
+                model: walletV2Model.collectiblesView.getAssetsList(root.slug)
+                StatusRoundedImage {
+                    id: image
+                    width: 146
+                    height: 146
+                    radius: 16
+                    image.source: model.imageUrl
+                    border.color: Theme.palette.baseColor2
+                    border.width: 1
+                    showLoadingIndicator: true
+                    color: model.backgroundColor
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: {
+                            openCollectibleDetailView({collectibleImageUrl:collectionImageUrl,
+                                                          name: model.name,
```

Comment:
nit: indent problem

Reply:
that's just qtCreators indentation. should be okay?